tests/functional/test_macro_resolve.js

```diff
@@ -0,0 +1,35 @@
+import { expect } from 'chai';
+import * as babel from '@babel/core';
+import dedent from 'dedent';
+
+const options = {
+    plugins: ['macros'],
+    filename: __filename,
+};
+
+describe('Macro resolve', () => {
+    before(() => {
+        // FIXME: mock `cosmiconfig`
```

Comment:
@AlexMost I tried several ways to mock `cosmiconfig` in mocha, but failed. 😂
Might need your help to take a look for this part.

Reply:
cjies#1